lib/ObjWriter/objwriter.cpp

```diff
@@ -599,19 +693,34 @@ extern "C" void EmitDebugFunctionInfo(ObjectWriter *OW,
     // This is the important bit that tells the debugger where the function
     // code is located and what's its size:
     OST.EmitIntValue(FunctionSize, 4);
-    OST.EmitFill(12, 0);
+    OST.EmitFill(4, 0);                // SS_DBGSTART
```

Comment:
Well. You are now in here. Can you use "ProcSym" to emit function symbol instead of emitting individual bytes like this? Then we don't need to add bunch of comments like this.